hypertrace-graphql-spaces-schema/src/main/java/org/hypertrace/graphql/spaces/dao/SpaceConfigRulesResponseConverter.java

```diff
@@ -0,0 +1,96 @@
+package org.hypertrace.graphql.spaces.dao;
+
+import io.reactivex.rxjava3.core.Maybe;
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.core.Single;
+import java.util.List;
+import javax.inject.Inject;
+import lombok.Value;
+import lombok.experimental.Accessors;
+import lombok.extern.slf4j.Slf4j;
+import org.hypertrace.core.graphql.common.utils.attributes.AttributeScopeStringTranslator;
+import org.hypertrace.graphql.spaces.schema.query.SpaceConfigRuleResultSet;
+import org.hypertrace.graphql.spaces.schema.shared.SpaceConfigRule;
+import org.hypertrace.graphql.spaces.schema.shared.SpaceConfigRuleAttributeValueRule;
+import org.hypertrace.graphql.spaces.schema.shared.SpaceConfigRuleType;
+import org.hypertrace.spaces.config.service.v1.AttributeValueRuleData;
+import org.hypertrace.spaces.config.service.v1.CreateRuleResponse;
+import org.hypertrace.spaces.config.service.v1.GetRulesResponse;
+
+@Slf4j
```

Comment:
Is this a shortcut for initializing a logger named after the class?

Reply:
Example:
```java
@Slf4j 
public class LogExample {}
```
 
will generate:
```java
 public class LogExample {
     private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LogExample.class);
 }
```

Reply:
very cool